awx/api/views/instance_install_bundle.py

```diff
@@ -49,9 +53,9 @@ class InstanceInstallBundle(GenericAPIView):
     def get(self, request, *args, **kwargs):
         instance_obj = self.get_object()
 
-        if instance_obj.node_type not in ('execution',):
+        if instance_obj.node_type not in ('execution', 'hop'):
```

Comment:
it would be good to make use of the enum objects instead of bare strings, like we do elsewhere

Reply:
- [ ] use enum where possible

Reply:
Yeah, I think this is still an open item but only a minor code style thing.